changelog/unreleased/breaking-changes/2949--pass-global-options-to-plugins-during-initialization.md

```diff
@@ -0,0 +1,5 @@
+There is a small addition to the plugin interface - the plugin initialization
+in `initialize()` now uses two `record` parameters. The former contains all
+plugin-relevant, the latter contains absolutely all configuration options. The
+plugins can now use both parameters for read-only access to all options in
+VAST.
```

Comment:
I don't think this is user-facing, so this doesn't warrant a changelog entry. We don't do dev-facing changelog entries currently.